templates/_connections.html.ep

```diff
@@ -1,39 +1,48 @@
-<div><table class="striped"><tbody>
+<ul class="collection departures connections">
 	% for my $res (@{$connections}) {
 		% my ($train, $via, $via_arr, $load) = @{$res};
 		% $via_arr = $via_arr ? $via_arr->strftime('%H:%M') : q{};
-		% my $td_class = '';
+		% my $row_class = '';
 		% my $link_class = 'action-checkin';
 		% if ($train->is_cancelled) {
-			% $td_class = 'cancelled';
+			% $row_class = 'cancelled';
 			% $link_class = 'action-cancelled-from';
 		% }
 		% if ($checkin_from) {
-			<tr class="<%= $link_class %>" data-station="<%= $train->station_uic %>" data-train="<%= $train->train_id %>" data-ts="<%= ($train->sched_departure // $train->departure)->epoch %>" data-dest="<%= $via->{name} %>">
+			<li class="collection-item <%= $row_class %> <%= $link_class %>"
+			data-station="<%= $train->station_uic %>"
+			data-train="<%= $train->train_id %>"
+			data-ts="<%= ($train->sched_departure // $train->departure)->epoch %>"
+			data-dest="<%= $via->{name} %>">
 		% }
 		% else {
-			<tr>
+			<li class="collection-item">
 		% }
-			<td class="<%= $td_class %>">
-				% if ($train->platform) {
-					% if ($checkin_from) {
-						<a>Gleis <%= $train->platform %></a>
-					% }
-					% else {
-						Gleis <%= $train->platform %>
-					% }
-					<br/>
-				% }
-				% if ($checkin_from) {
-					<a><%= $train->line %></a>
+			<a class="dep-time" href="#">
+				% if ($train->departure_is_cancelled) {
+					%= $train->sched_departure->strftime('%H:%M')
 				% }
 				% else {
-					%= $train->line
+					%= $train->departure->strftime('%H:%M')
+				% }
+				% if ($via_arr) {
+					→ <%= $via_arr %>
 				% }
-			</td>
-			<td class="<%= $td_class %>">
-				% if ($checkin_from) {
-					<a><%= $via->{name} %></a>
+				% if ($train->departure_delay) {
+					%= sprintf('(%+d)', $train->departure_delay)
+				% }
+			</a>
+			<span class="connect-platform-wrapper">
+				% if ($train->platform) {
+					<span>Gleis <%= $train->platform %></span>
+				% }
+				<span class="dep-line <%= (split(/ /, $train->line))[0] %>">
```

Comment:
Note: there are trains that do not have a line type, e.g. in Belgium. In that case, this piece of code will return the train number instead, which is not helpful but also not harmful.

I'll quickly replace it with `$train->type` here and in the other three cases.